test/wampy-serializer-handshake-test.js

```diff
@@ -1,60 +1,49 @@
-
-/**
- * Project: wampy.js
- * User: KSDaemon
- * Date: 13.06.17
- */
 import { expect } from 'chai';
 import { WebSocket, setProtocol as wsSetProtocol } from './fake-ws-set-protocol.js';
 import { Wampy } from './../src/wampy.js';
 import { JsonSerializer } from '../src/serializers/JsonSerializer.js';
 import { CborSerializer } from '../src/serializers/CborSerializer.js';
-import * as Errors from '../src/errors.js';
-
-const routerUrl = 'ws://fake.server.org/ws/';
+import { NoSerializerAvailableError } from '../src/errors.js';
 
 describe('Wampy.js Serializer Handshake', function () {
     this.timeout(0);
 
-    it('disallows to connect when server choose not available serializer', async function () {
-        let wampy = new Wampy(routerUrl, {
-            realm: 'AppRealm',
-            ws: WebSocket,
-            serializer: new JsonSerializer()
-        });
-        wsSetProtocol('cbor');
+    const testUrl = 'ws://fake.server.org/ws/';
+    const defaultOptions = { realm: 'AppRealm', ws: WebSocket };
+    const jsonSerializerOptions = { ...defaultOptions, serializer: new JsonSerializer() };
```

Comment:
I think this name is a bit wrong. It can be treated like options for json serializer but in reality it is config for wampy instance itself with json serializer as serializer. So someone who is not so familiar with wampy, having look at the code `const wampy = new Wampy(testUrl, jsonSerializerOptions);` can not understand it correctly.
Same applies for cbor variant.

Reply:
What do you think of "wampyOptionsWithJsonSerializer" ?

Reply:
👍🏻